docs/reference/ccr/index-following.asciidoc

```diff
@@ -0,0 +1,98 @@
+[role="xpack"]
+[testenv="platinum"]
+[[index-following]]
+=== Index following
+
+Index following is the core feature provided by {ccr}. You can use the
+{ref}/ccr-put-follow.html[create follower API] to create a follower index and
+initiate the following of a leader index on a remote cluster.
+
+=== Initiating following
+
+When you submit a {ref}/ccr-put-follow.html[create follower request], the follower
+index is created on the follower cluster. Once the follower index is created,
```

Comment:
We try to avoid notions such as "follower cluster" and "leader cluster" since these aren't real concepts, as replication is done at the index level, and we can have replication one way on one index, and the reverse way on another index. We have been using the terminology "local" and "remote".